### ISSUE TITLE
Empty ModifierVisitorAdapter causing CompilationUnit to throw NPE in toString()

### ISSUE THREAD

**Issue by ide** [javaparser/javaparser#391]
I upgraded to javaparser 2.5.0 and am getting a NullPointerException with my ModifierVisitorAdapter. I deleted a bunch of code down to a simple repro case:

Transformer code:

``` java
FileInputStream in = new FileInputStream(path);
CompilationUnit cu = JavaParser.parse(in);
in.close();

(new ModifierVisitorAdapter() {}).visit(cu, null);

String output = cu.toString();
```

String to parse and modify:

``` java
public class Example {
  private String mString;
  public Example(String arg) {
    mString = arg;
  }
}
```

This throws:

```
Exception in thread "main" java.lang.NullPointerException
    at com.github.javaparser.ast.visitor.DumpVisitor.visit(DumpVisitor.java:696)
    at com.github.javaparser.ast.expr.AssignExpr.accept(AssignExpr.java:87)
    at com.github.javaparser.ast.visitor.DumpVisitor.visit(DumpVisitor.java:1195)
    at com.github.javaparser.ast.stmt.ExpressionStmt.accept(ExpressionStmt.java:65)
    at com.github.javaparser.ast.visitor.DumpVisitor.visit(DumpVisitor.java:1170)
    at com.github.javaparser.ast.stmt.BlockStmt.accept(BlockStmt.java:68)
    at com.github.javaparser.ast.visitor.DumpVisitor.visit(DumpVisitor.java:1030)
    at com.github.javaparser.ast.body.ConstructorDeclaration.accept(ConstructorDeclaration.java:105)
    at com.github.javaparser.ast.visitor.DumpVisitor.printMembers(DumpVisitor.java:204)
    at com.github.javaparser.ast.visitor.DumpVisitor.visit(DumpVisitor.java:391)
    at com.github.javaparser.ast.body.ClassOrInterfaceDeclaration.accept(ClassOrInterfaceDeclaration.java:97)
    at com.github.javaparser.ast.visitor.DumpVisitor.visit(DumpVisitor.java:297)
    at com.github.javaparser.ast.CompilationUnit.accept(CompilationUnit.java:91)
    at com.github.javaparser.ast.Node.toString(Node.java:290)
```

Either a more descriptive error message or more leniency in the visitor would make this easier to debug.

**Comment by DeepSnowNeeL** [comment]
It might be due to my modifications, I'll check this out

**Comment by peacepassion** [comment]
I encounter the same NPE.

**Comment by DeepSnowNeeL** [comment]
Allright fixed it, Check #392

**Comment by ide** [comment]
😮  thank you!

**Comment by matozoid** [comment]
Alright, it's released as 2.5.1
